runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -118,17 +122,45 @@ private void translateKafkaInput(
     }
 
     final String topic;
-    Preconditions.checkNotNull(topic = (String) params.get("topic"), "'topic' needs to be set");
+    final String groupId;
+    final String bootstrapServers;
 
+    Preconditions.checkNotNull(topic = (String) params.get("topic"), "'topic' needs to be set");
     Map<?, ?> consumerProps = (Map) params.get("properties");
     Preconditions.checkNotNull(consumerProps, "'properties' need to be set");
-    final Properties properties = new Properties();
-    properties.putAll(consumerProps);
 
-    logger.info("Kafka consumer for topic {} with properties {}", topic, properties);
+    logger.info("Configuring kafka consumer for topic {} with properties {}", topic, consumerProps);
+
+    Preconditions.checkNotNull(
+        groupId = (String) consumerProps.remove("group.id"), "'group.id' needs to be set");
+    Preconditions.checkNotNull(
+        bootstrapServers = (String) consumerProps.remove("bootstrap.servers"),
+        "'bootstrap.servers' needs to be set");
+
+    final String splits[] = bootstrapServers.split(":", 2);
```

Comment:
```suggestion
    final String bootstrapServers[] = bootstrapServersString.split(":", 2);
```

Reply:
Does this code also work if there are multiple bootstrap servers and some of them do not supply a port?

Reply:
Currently, the setup at Lyft is to just pass along the cluster name and not specific brokers.  Looks like the cluster name we pass is a LB (3) 

1.https://github.com/lyft/streamingcontroller/blob/master/kafkatopicprovisioner/clusters/cluster-tenant-map.yaml#L145
2. https://github.com/lyft/pricingrealtime/blob/master/ops/sp/pricingrealtime-production.jsonnet#L11   
3. https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:search=prod-iad-1;sort=loadBalancerName

Having said that, I agree that if users get fancy and pass along the actual brokers, the above code will fail as it is baked with a certain weak assumption. The plan is to move this to the builder code.

Reply:
Code is refactored to honor multiple bootstrap servers being passed in properties.

Reply:
Do we test multiple servers in the builder? Otherwise we should probably add a test in this PR.